src/components/projectSelect/ProjectOptionGroup.tsx

```diff
@@ -37,9 +39,21 @@ export const ProjectOptionGroup: React.FC<OptionGroupProps> = ({
   name,
   projects,
   onClick,
+  repoIdentifier,
+  canClickOnRepoGroup = false,
 }) => (
   <OptionGroupContainer>
-    <Overline>{name}</Overline>
+    {/* if it's the project settings page and it's not the "" group, make the header clickable */}
+    {canClickOnRepoGroup ? (
+      <OverlineHover>
+        <Overline role="button" onClick={() => onClick(repoIdentifier)}>
+          {name}
+        </Overline>
+      </OverlineHover>
```

Comment:
On smaller project names the clickable area is difficult to select.
<img width="172" alt="image" src="https://user-images.githubusercontent.com/4605522/157557128-97b2aa5a-e104-460b-9f4a-6078c55526e3.png">

Reply:
I added some padding to the hover.